lib/contentful/client.rb

```diff
@@ -30,8 +30,11 @@ class Client
       proxy_host: nil,
       proxy_username: nil,
       proxy_password: nil,
-      proxy_port: nil
+      proxy_port: nil,
+      max_rate_limit_retries: 1
     }
+    # Rate Limit Reset Header Key
+    RATE_LIMIT_RESET_HEADER_KEY = 'x-contentful-ratelimit-reset'
```

Comment:
I'm guessing accessing the headers is case insensitive in whatever HTTP library you're using? Because the headers are capitalized :)